src/main/java/seedu/duke/tasks/Task.java

```diff
@@ -0,0 +1,124 @@
+package seedu.duke.tasks;
+
+public class Task {
+    protected final String description;
+    protected final String dateTime;
+    protected final boolean isDone;
```

Comment:
If you decide to make a Task mutable, remove `final` from these variables. Also, I think these should be `private` to the `Task` class, and shouldn't be accessible within the class (perhaps have a getter class to return these values).